source/projects/private-voice-assistant/voice_remote_esp32.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: "Setting up a $13 voice remote to control Home Assistant"
+---
+
+This tutorial will guide you to turn an ATOM Echo into the
+world's most private voice assistant. Pick up the tiny device to talk to
+your smart home. Issue commands and get responses!
+
+* Don't have Home Assistant Cloud? You just want fully local processing? Follow this procedure: [Setting up a fully local voice remote device](/projects/private-voice-assistant/voice_remote_local_device/)
```

Comment:
We should link this video instead:

```suggestion
<lite-youtube videoid="w6QxGdxVMJs" videotitle="$13 voice remote for Home Assistant
"></lite-youtube>
```

The local pipeline suggestion should be on the `assist_pipeline` docs, which will be linked in requirements.